droidlet/lowlevel/hello_robot/remote/launch_ros.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/env bash
+set -e
+
+if $(which python | grep fairo >/dev/null); then
+    echo "droidlet virtualenv needs to be deactivated before running this script. Run 'deactivate' in the terminal before running this"
```

Comment:
will this cause issues?

Reply:
not really. it's a check to make sure issues aren't caused by accidentally forgetting to do `deactivate` or having a virtualenv active